File: code_combat/7_Sarven_Desert/298-Odd_Sandstorm/oddSandstorm.js
// This array contains friends and ogres.
// The even elements are ogres, but the odds are friends.
var everybody = ['Yetu', 'Tabitha', 'Rasha', 'Max', 'Yazul',  'Todd'];
var enemyIndex = 0;

while (enemyIndex < everybody.length) {
    // Use square brackets to get the ogre name from the array.
    var ogre = everybody[enemyIndex];
    // Attack using the variable holding the ogre name.
    if (hero.isReady("bash")) hero.bash(ogre);
    else hero.attack(ogre);
    // Increment by 2 to skip over friends.
    enemyIndex += 2;
}

// After defeating ogres, move to the oasis.
hero.moveXY(35, 54);

File: projects/code_combat/7_Sarven_Desert/298-Odd_Sandstorm/odd_sandstorm.py
everybody = ['Yetu', 'Tabitha', 'Rasha', 'Max', 'Yazul',  'Todd']
enemyIndex = 0

while enemyIndex < everybody.length:
    ogre = everybody[enemyIndex]
    hero.attack(ogre)
    enemyIndex += 2

hero.moveXY(35, 54)
